translations/en.json

```diff
@@ -30,8 +30,8 @@
   "onHoldStatus": "ON_HOLD",
   "orderDate": "Order Date:",
   "Ordered tests": "Ordered tests",
-  "ordererName": "Orderer Name: ",
-  "orderInStruction": "Instructions: ",
+  "orderedBy": "Ordered By: ",
+  "orderInstructions": "Instructions: ",
```

Comment:
Let's remove the trailing space at the end of the string as well so that there no unnecessary spaces in translation strings